pkgs/applications/networking/geph5/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libxkbcommon,
+  openssl,
+  jemalloc,
+  sqlite,
+  vulkan-loader,
+  wayland,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "geph5";
+  version = "0.2.55";
+
+  src = fetchFromGitHub {
+    owner = "geph-official";
+    repo = "geph5";
+    rev = "geph5-client-v${finalAttrs.version}";
+    hash = "sha256-oq/aPh60QtBQ+tdX7j1NaCCe8G5THsrqAd4Is4yc9fA=";
+  };
+
+  cargoHash = "sha256-jzjCTQRhUr2Eif6/AzZCZSrKjNbJdR60qgZ6xQ87B1w=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    libxkbcommon
+    openssl
+    jemalloc
+    sqlite
+    vulkan-loader
+    wayland
+  ];
+
+  cargoBuildFlags = [
+    "--package"
+    "geph5-client"
+  ];
+  doCheck = false;
+
+  env = {
+    OPENSSL_NO_VENDOR = true;
+  };
+
+  meta = {
+    description = "Modular Internet censorship circumvention system designed specifically to deal with national filtering";
+    homepage = "https://github.com/geph-official/geph5";
+    changelog = "https://github.com/geph-official/geph5/releases/tag/geph5-client-v${finalAttrs.version}";
+    mainProgram = "geph5-client";
+    platforms = lib.platforms.linux;
```

Comment:
It seems there should be `unix`.
https://github.com/geph-official/geph5/blob/d793aa89e862e11464c46f5cc0ab1b6011215f07/binaries/geph5-client/src/vpn.rs#L30